DataFormats/L1TGlobal/interface/GlobalObject.h

```diff
@@ -23,11 +23,12 @@ enum GlobalObject
     gtETT,
     gtHTT,
     gtHTM,
-    gtETM2,
+    gtETMHF,
```

Comment:
@rekovic - this is a new name for the same thing? Or does this change introduce a backwards incompatibility (well, if we store this data format someplace)